content/error-handling/error-codes.textile

```diff
@@ -0,0 +1,301 @@
+---
+title: Error handling
+meta_description: " "
+---
+
+To resolve issues effectively, refer to the error codes provided. Each error code is supported by a description of the problem, examples of when it might occur, and steps to address it.
+
+If the provided information does not resolve your issue, contact Ably "support":https://ably.com/support. When contacting, include details such as your app ID, the error code, and any relevant logs to help troubleshoot. Include as much of the following information as possible:
+* Provide timestamps in UTC format.
+* Include complete SDK logs from the time of the failure. Ensure these logs show activity before and after the timeout, as SDKs retry failed requests by default.
+* Specify which endpoints were accessed. Mention if you use custom client options, the environment setting, and the failing SDK operation.
+* State the SDK/s you use, including the platform and versions.
+* Include any stack traces related to the error.
+* Indicate whether the issue occurs consistently or was a one-time event.
+* Provide details to confirm whether the issue was related to your network or Ably's availability. For example, note whether other internet operations were succeeding simultaneously.
+* Include the app ID associated with the request.
```

Comment:
I think this would be better off on the main errors page, rather than here. WDYT?